pkg/apis/local/v1/types.go

```diff
@@ -37,6 +37,9 @@ type LocalVolumeSpec struct {
 	NodeSelector *corev1.NodeSelector `json:"nodeSelector,omitempty"`
 	// List of storage class and devices they can match
 	StorageClassDevices []StorageClassDevice `json:"storageClassDevices,omitempty"`
+	// If specified, a list of tolerations to pass to the diskmaker and provisioner DaemonSets.
+	// +optional
+	Tolerations []corev1.Toleration `json:"tolerations,omitempty" protobuf:"bytes,22,opt,name=tolerations"`
```

Comment:
Is `protobuf` required here? We're not using this tag in any other field.

Reply:
I initially copied it from the Pod definition. Testing with it removed reveals no difference that I've noticed. The e2e test continues to pass, and if I test manually it still schedules on the tainted node.

I've removed this bit for now.